BGL/include/CGAL/boost/graph/helpers.h

```diff
@@ -751,6 +751,20 @@ void clear(FaceGraph& g)
   CGAL_postcondition(num_faces(g) == 0);
 }
 
+
+template <typename FaceGraph>
+typename boost::graph_traits<FaceGraph>::degree_size_type
+face_degree(typename boost::graph_traits<FaceGraph>::halfedge_descriptor hd, const FaceGraph& g)
+{
+  typename boost::graph_traits<FaceGraph>::degree_size_type res = 0;
+  typename boost::graph_traits<FaceGraph>::halfedge_descriptor done = hd;
+  do {
+    ++res;
+    hd = next(hd,g);
+  }while(! (hd == done));
+  return res;
+}
```

Comment:
In fact it should be called just `degree()`. I think we did it call this way and had a problem with sfinae. Note that the PR #2101 introduces implementations of this function in the models, as it is listed in the concept.  One might (should??) argue that the degree functions should only be generic, as it is nothing but counting `next()` steps around a face, or `opposite(next())` steps around a vertex.